packages/algob/src/lib/tx.ts

```diff
@@ -266,20 +268,70 @@ export async function makeAndSignTx (
   return [txns, signedTxn];
 }
 
+/**
+ * Signs transaction object(s) and returns raw signed transaction
+ * Note: Sign transaction is used to sign single transaction and `signTransactions` takes
+ * SDK transaction object, signer and signs it.
+ * @param txnAndSign Transaction object(s) with signers
+ */
+export function signTransactions (
+  txnAndSign: wtypes.TransactionAndSign[]
+): Uint8Array | Uint8Array[] {
+  let txns: Transaction[] = [];
+  const signers: wtypes.Sign[] = [];
+  txnAndSign.forEach(function (value, idx) {
+    txns[idx] = value.transaction;
+    signers[idx] = value.sign;
+  });
```

Comment:
let's use for loop instead of `foreach`.

Reply:
is there any specific reason ?

Reply:
This comment is not blocking (you can merge). Having a function to modify outside variables is not very clear.

Reply:
ok, updating to `for`